packages/components/src/components/select/select.ts

```diff
@@ -97,6 +97,8 @@ export default class SdSelect extends SolidElement implements SolidFormControl {
    */
   @state() private displayLabel = '';
   /** @internal */
+  @state() private liveLabel = '';
```

Comment:
I feel it would be better to give it a more descriptive name, something like `deletedTagLabel`.